packages/cli-kit/project.json

```diff
@@ -21,6 +21,22 @@
           "cwd": "packages/cli-kit"
         }
       },
+      "docs:generate": {
+        "executor": "nx:run-commands",
+        "outputs": ["docs"],
+        "options": {
+          "command": "pnpm typedoc --out docs $(find ./src/public/**/{*.ts,*.tsx} | grep -v .test.ts)",
```

Comment:
I'm not sure this syntax works across all shells: can you try if this command does the same thing? `find ./src/public -name "*.test.ts" | xargs pnpm typedoc --out docs`

Reply:
Not from Shopify, but we want the community to be able to contribute